### PR TITLE
Feature/151 add form id manually if no auth

### PR DESCRIPTION
**Pull request by wayangalihpratama**
### TODO / DONE

---

- [x] Modify AuthByPassForm component to add form id input
- [x] Add Add Form menu in setting (only shown if no code_assignment in auth type build params)
- [x] Create new page to handle Add New form

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205434138509512

### CODE REVIEW

**Review by dedenbangkit — CHANGES_REQUESTED** [review]
Please update the test @wayangalihpratama

**Comment by wayangalihpratama** [comment]
> Please update the test @wayangalihpratama

Hi mas @dedenbangkit , 
I have updated the test.

File: app/src/build.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "authenticationType": ["code_assignment", "username", "password"],
+  "authenticationType": ["username", "password"],
```
Comment (dedenbangkit) [inline app/src/build.json]: please don't change this, this will causing error for nwmis release

**Review by dedenbangkit — CHANGES_REQUESTED** [review]

File: app/src/build.json (hunk 1 above)
Comment (dedenbangkit) [reply app/src/build.json]: you need to mock the buildparams

Comment (wayangalihpratama) [reply app/src/build.json]: I have revert above change mas @dedenbangkit

**Pull request merged**